src/condvar.rs

```diff
@@ -197,11 +195,10 @@ impl Condvar {
 
     #[cold]
     fn notify_all_slow(&self, mutex: *mut RawMutex) -> usize {
-        unsafe {
             // Unpark one thread and requeue the rest onto the mutex
             let from = self as *const _ as usize;
             let to = mutex as usize;
-            let validate = || {
+            let validate = unsafe { || {
```

Comment:
Can this `unsafe` be moved inside the closure?

Reply:
Sorry for the long reply, very good suggestion